Content.Client/Store/Ui/StoreMenu.xaml.cs

```diff
@@ -77,10 +70,12 @@ public void UpdateBalance(Dictionary<string, FixedPoint2> balance)
         WithdrawButton.Disabled = disabled;
     }
 
-    public void UpdateListing(List<ListingData> listings)
+    public void UpdateListing(List<ListingData>? listings = null)
     {
-        var sorted = listings.OrderBy(l => l.Priority).ThenBy(l => l.Cost.Values.Sum());
+        if (listings != null)
+            _cachedListings = listings;
 
+        var sorted = _cachedListings.OrderBy(l => l.Priority).ThenBy(l => l.Cost.Values.Sum());
```

Comment:
This api isn't great, just add an empty override instead of passing in null to indicate no change.